config-provider-library/src/main/java/com/aevi/sdk/pos/flow/config/SettingsProvider.java

```diff
@@ -4,37 +4,29 @@
 import android.content.SharedPreferences;
 import android.preference.PreferenceManager;
 
+import com.aevi.sdk.flow.model.config.AppFlowSettings;
 import com.aevi.sdk.flow.model.config.FpsSettings;
-import com.aevi.sdk.pos.flow.config.model.Channels;
 
 import javax.inject.Inject;
 import javax.inject.Singleton;
 
 import io.reactivex.subjects.PublishSubject;
 
+import static com.aevi.sdk.flow.constants.config.FlowConfigKeys.*;
+
 @Singleton
 public class SettingsProvider {
 
     private final SharedPreferences preferences;
 
-    private static final String KEY_FPS_SETTINGS = "fpsSettings";
     private static final String KEY_AUTO_GENERATE_CONFIGS = "autoGenerateConfigs";
     private static final String KEY_OVERWRITE_CONFIGS_ON_REINSTALL = "overwriteConfigsOnReinstall";
-    private static final String KEY_COMMS_CHANNEL = "commsChannel";
 
     private PublishSubject<SettingsProvider> settingsChangeSubject = PublishSubject.create();
 
-    private FpsSettings fpsSettings;
-
     @Inject
     public SettingsProvider(Context context) {
         preferences = PreferenceManager.getDefaultSharedPreferences(context);
```

Comment:
This should probably be in the dagger module so we can unit test by mocking the SharedPreferences